perfmetrics/scripts/job_files/seq_rand_read_write.fio

```diff
@@ -24,267 +24,124 @@ startdelay=300
-filesize=3M
-numjobs=10
-
-[7_thread]
-stonewall
-startdelay=2520
 directory=gcs/3mb
 filesize=3M
 rw=write
-numjobs=10
-
-[8_thread]
-stonewall
-startdelay=2890
-directory=gcs/3mb
-filesize=3M
-rw=write
-numjobs=20
-
-[9_thread]
-stonewall
-startdelay=3260
-directory=gcs/5mb
-filesize=5M
-numjobs=10
-
-[10_thread]
-stonewall
-startdelay=3630
-directory=gcs/5mb
-filesize=5M
-numjobs=20
-
-[11_thread]
-stonewall
-startdelay=4000
-directory=gcs/5mb
-filesize=5M
 numjobs=40
 
-[12_thread]
+[5_thread]
 stonewall
-startdelay=4370
+startdelay=1780
 directory=gcs/5mb
 filesize=5M
-rw=write
 numjobs=40
 
-[13_thread]
+[6_thread]
 stonewall
-startdelay=4740
+startdelay=2150
 directory=gcs/5mb
 filesize=5M
 rw=write
-numjobs=10
-
-[14_thread]
-stonewall
-startdelay=5110
-directory=gcs/50mb
-filesize=50M
 numjobs=40
 
-[15_thread]
-stonewall
-startdelay=5480
-directory=gcs/50mb
-filesize=50M
-numjobs=20
-
-[16_thread]
+[7_thread]
 stonewall
-startdelay=5850
+startdelay=2520
 directory=gcs/50mb
 filesize=50M
-rw=write
 numjobs=40
 
-[17_thread]
+[8_thread]
 stonewall
-startdelay=6220
+startdelay=2890
 directory=gcs/50mb
 filesize=50M
 rw=write
-numjobs=10
-
-[18_thread]
-stonewall
-startdelay=6590
-bs=16k
-directory=gcs/256kb
-filesize=256k
-rw=randread
-numjobs=20
+numjobs=40
 
-[19_thread]
+[9_thread]
 stonewall
-startdelay=6960
+startdelay=3260
 bs=16k
 directory=gcs/256kb
 filesize=256k
 rw=randread
 numjobs=40
 
-[20_thread]
+[10_thread]
 stonewall
-startdelay=7330
+startdelay=3630
 bs=16k
 directory=gcs/256kb
 filesize=256k
 rw=randwrite
 numjobs=40
 
-[21_thread]
-stonewall
-startdelay=7700
-bs=16k
-directory=gcs/256kb
-filesize=256k
-rw=randwrite
-numjobs=10
-
-[22_thread]
-stonewall
-startdelay=8070
-directory=gcs/3mb
-filesize=3M
-rw=randread
-numjobs=20
-
-[23_thread]
+[11_thread]
 stonewall
-startdelay=8440
+startdelay=4000
 directory=gcs/3mb
 filesize=3M
 rw=randread
-numjobs=10
-
-[24_thread]
-stonewall
-startdelay=8810
-directory=gcs/3mb
-filesize=3M
-rw=randwrite
-numjobs=10
+numjobs=40
 
-[25_thread]
+[12_thread]
 stonewall
-startdelay=9180
+startdelay=4370
 directory=gcs/3mb
 filesize=3M
 rw=randwrite
-numjobs=20
-
-[26_thread]
-stonewall
-startdelay=9550
-directory=gcs/5mb
-filesize=5M
-rw=randread
-numjobs=10
-
-[27_thread]
-stonewall
-startdelay=9920
-directory=gcs/5mb
-filesize=5M
-rw=randread
-numjobs=20
+numjobs=40
 
-[28_thread]
+[13_thread]
 stonewall
-startdelay=10290
+startdelay=4740
 directory=gcs/5mb
 filesize=5M
 rw=randread
 numjobs=40
 
-[29_thread]
+[14_thread]
 stonewall
-startdelay=10660
+startdelay=5510
 directory=gcs/5mb
 filesize=5M
 rw=randwrite
 numjobs=40
 
-[30_thread]
-stonewall
-startdelay=11030
-directory=gcs/5mb
-filesize=5M
-rw=randwrite
-numjobs=10
-
-[31_thread]
+[15_thread]
 stonewall
-startdelay=11400
+startdelay=5480
 directory=gcs/50mb
 filesize=50M
 rw=randread
 numjobs=40
 
-[32_thread]
-stonewall
-startdelay=11770
-directory=gcs/50mb
-filesize=50M
-rw=randread
-numjobs=20
-
-[33_thread]
+[16_thread]
```

Comment:
this is the job name. name should convey the job . 16_thread has nothing to do with the job config here. can you rename it to write_50mb and saame for other jobs as well.

Reply:
merging it. we will change the name later.